cloudos/__main__.py

```diff
@@ -64,7 +64,13 @@ def cromwell():
               help=('A config file similar to a nextflow.config file, ' +
                     'but only with the parameters to use with your job.'))
 @click.option('-p',
-              '--nextflow-profile',
+              '--parameter',
+              multiple=True,
+              help=('A single parameter to pass to the job call. It should be in the ' +
+                    'following form: parameter_name=parameter_value. E.g.: ' +
+                    '-p input=s3://path_to_my_file. You can use this option as many ' +
+                    'times as parameters you want to include.'))
+@click.option('--nextflow-profile',
```

Comment:
nice now only `--nextflow-profile` 👍 will be used.
We are technically backwards incompatible now because if a user has a command in their notes with `-p` meaning profile it would break, we could bump to 1.0.0, a new major release to denote that. But it's ok, for now, let's skip it since the feature was experimental.